src/components/program-progress/data/utils.jsx

```diff
@@ -173,3 +179,22 @@ export function hasLicenseOrCoupon() {
 
   return hasActiveLicenseOrLicenseRequest || hasOffersOrCouponCodeRequests;
 }
+
+export const courseUpgradationAvailable = (course) => course.upgradeUrl
+  && !(course.expired === true)
```

Comment:
nit: can this condition be changed to `!(course.expired === true)` -> `!course.expired`?